<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>102</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="checkboxes">
     <item>
      <widget class="QCheckBox" name="checkBox1">
       <property name="text">
        <string>CheckBox 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox2">
       <property name="text">
        <string>CheckBox 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox3">
       <property name="text">
        <string>CheckBox 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox4">
       <property name="text">
        <string>CheckBox 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox5">
       <property name="text">
        <string>CheckBox 5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="mywidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>541</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
